examples/browser_on.js

```diff
@@ -0,0 +1,41 @@
+import launcher from 'k6/x/browser';
```

Comment:
Currently it's only possible to add E2E tests for this feature, since [the mock `VU` doesn't have a `RegisterCallback` implementation yet](https://github.com/grafana/k6/blob/c31e840793b392e16316d35b83e08345629b2fd5/js/modulestest/modulestest.go#L62-L66), so we can't have a Go integration test for it.

If this is a blocker for merging this, then we probably should contribute that to k6... :sweat_smile:

Reply:
:) I think we can merge this first and create another issue for the E2E test (if mock VU `RegisterCallback` won't be available yet).

Reply:
I'll make a note in #95 and leave the issue open so that we don't forget to add tests for it once https://github.com/grafana/k6/pull/2399 is merged.

But I think we can merge this PR with this single E2E test.